src/server/response.cpp

```diff
@@ -31,8 +31,10 @@
 #include <mustache.hpp>
 #include <zlib.h>
 
+#include <array>
 
-#define KIWIX_MIN_CONTENT_SIZE_TO_DEFLATE 100
+
+#define KIWIX_MIN_CONTENT_SIZE_TO_COMPRESS 100
```

Comment:
I wonder if this threshold is too low - why trying to compress payload that is smaller than the header section?

Reply:
Probably yes. It is a pretty old magic value and it seems it has been introduce as some kind of workaround for a segfault https://github.com/kiwix/kiwix-tools/commit/4c79cfcf06e75ef4c38fd45649a36ed426e80445

The question is what would be the good value ? I've seen different values but it seems that MTU is a good limit (https://www.computerworld.com/article/2693941/why-it-doesn-t-make-sense-to-gzip-all-content-from-your-web-server.html). So we can move this to 1400 Bytes.